modules/Router.js

```diff
@@ -15,5 +15,48 @@
    * @class  Router
    * @author kent.willis@vecna.com
    */
-  return Backbone.Router.extend({});
+
+  var Router = Backbone.Router.extend({
+
+    /**
+    * overridden the route function to send start and end times to Logger
+    */
+    route: function(route, name, callback) {
+      if (!_.isRegExp(route)) route = this._routeToRegExp(route);
+      if (_.isFunction(name)) {
+        callback = name;
+        name = '';
+      }
+      if (!callback) callback = this[name];
+      var router = this;
+      Backbone.history.route(route, function(fragment) {
+
+        newrelic.setCustomAttribute('route', name);
+        var UUID = "uuid-"+(new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
```

Comment:
Can you elaborate on this? I'm not sure how to make these overrides mixins. For a number of the overridden methods, I'm modifying the callback functions themselves, so I'm changing internal sections of the overridden functions. Is there a way to do this (with or without using mixins) that doesn't require copying and pasting methods from backbone?